docs/source/backends/adios2.rst

```diff
@@ -0,0 +1,77 @@
+.. _backends-adios2:
+
+ADIOS2 Backend
+==============
+
+openPMD supports writing to and reading from ADIOS2 ``.bp`` files.
+For this, the installed copy of openPMD must have been built with support for the ADIOS2 backend.
+To build openPMD with support for ADIOS2, use the CMake option ``-DopenPMD_USE_ADIOS2=ON``.
+For further information, check out the :ref:`installation guide <install>`,
+:ref:`build dependencies <development-dependencies>` and the :ref:`build options <development-buildoptions>`.
+
+
+I/O Method
+----------
+
+ADIOS2 has several engines for alternative file formats and other kinds of backends, yet natively writes to ``.bp`` files. At the moment, the openPMD API exclusively uses the BPFile engine.
+We currently leverage the default ADIOS2 transport parameters, i.e. ``POSIX`` on Unix systems and ``FStream`` on Windows.
+
+
+Backend-Specific Controls
+-------------------------
+
+The following environment variables control ADIOS2 I/O behavior at runtime.
+Fine-tuning these is especially useful when running at large scale.
+
+===================================== ======= ===================================================================
+environment variable                  default description
+===================================== ======= ===================================================================
+``OPENPMD_ADIOS2_HAVE_PROFILING``      ``1``   Turns on/off profiling information right after a run.
+``OPENPMD_ADIOS2_HAVE_METADATA_FILE``  ``1``   Online creation of the adios journal file (``1``: yes, ``0``: no).
```

Comment:
@franzpoeschel are you sure that is still correct?

By default, this now creates a `<name>.bp/` directory in ADIOS2 and `bpls` (among others) seams to be able to operate directly on that - without an extra metadata file.